[2025-05-22T01:37:23.758+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-22T01:37:23.770+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_sales_stream_transformation.transform_sales_stream manual__2025-05-21T20:07:20.807512+00:00 [queued]>
[2025-05-22T01:37:23.776+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_sales_stream_transformation.transform_sales_stream manual__2025-05-21T20:07:20.807512+00:00 [queued]>
[2025-05-22T01:37:23.777+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-05-22T01:37:23.791+0530] {taskinstance.py:2888} INFO - Executing <Task(SnowflakeSqlApiOperator): transform_sales_stream> on 2025-05-21 20:07:20.807512+00:00
[2025-05-22T01:37:23.794+0530] {standard_task_runner.py:72} INFO - Started process 168944 to run task
[2025-05-22T01:37:23.797+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'elt_sales_stream_transformation', 'transform_sales_stream', 'manual__2025-05-21T20:07:20.807512+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/elt_transformation_dag.py', '--cfg-path', '/tmp/tmp0n7npvw2']
[2025-05-22T01:37:23.798+0530] {standard_task_runner.py:105} INFO - Job 128: Subtask transform_sales_stream
[2025-05-22T01:37:23.824+0530] {task_command.py:467} INFO - Running <TaskInstance: elt_sales_stream_transformation.transform_sales_stream manual__2025-05-21T20:07:20.807512+00:00 [running]> on host Voldemort.
[2025-05-22T01:37:23.885+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='elt_sales_stream_transformation' AIRFLOW_CTX_TASK_ID='transform_sales_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T20:07:20.807512+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T20:07:20.807512+00:00'
[2025-05-22T01:37:23.886+0530] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-22T01:37:23.899+0530] {snowflake.py:412} INFO - Executing: 
        CREATE OR REPLACE TABLE AIRFLOW_DB.PUBLIC.SALES_STREAM_TRANSFORMED AS
        SELECT
            REGION,
            COUNTRY,
            "ITEM TYPE",
            "SALES CHANNEL",
            "ORDER PRIORITY",
            "ORDER DATE",
            "ORDER ID",
            "SHIP DATE",
            "UNITS SOLD",
            "UNIT PRICE",
            "UNIT COST",
            "TOTAL REVENUE",
            "TOTAL COST",
            "TOTAL PROFIT",
            CASE
                WHEN "ORDER PRIORITY" = 'H' THEN 'High'
                WHEN "ORDER PRIORITY" = 'L' THEN 'Low'
                WHEN "ORDER PRIORITY" = 'M' THEN 'Medium'
                WHEN "ORDER PRIORITY" = 'C' THEN 'Cold'
                ELSE "ORDER PRIORITY"
            END AS "ORDER_PRIORITY_DESC",
            DATEDIFF(
                'day',
                COALESCE(
                  TRY_TO_DATE("ORDER DATE", 'M/D/YYYY'),
                  TRY_TO_DATE("ORDER DATE", 'MM/DD/YYYY'),
                  TRY_TO_DATE("ORDER DATE", 'YYYY-MM-DD')
                ),
                COALESCE(
                  TRY_TO_DATE("SHIP DATE", 'M/D/YYYY'),
                  TRY_TO_DATE("SHIP DATE", 'MM/DD/YYYY'),
                  TRY_TO_DATE("SHIP DATE", 'YYYY-MM-DD')
                )
            ) AS "SHIPMENT_LEAD_TIME"
        FROM AIRFLOW_DB.PUBLIC."SALES STREAM";
        
[2025-05-22T01:37:23.907+0530] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-05-22T01:37:23.914+0530] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/providers/snowflake/operators/snowflake.py", line 420, in execute
    self.query_ids = self._hook.execute_query(
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/providers/snowflake/hooks/snowflake_sql_api.py", line 140, in execute_query
    conn_config = self._get_conn_params
  File "/usr/lib/python3.10/functools.py", line 981, in __get__
    val = self.func(instance)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 315, in _get_conn_params
    p_key = serialization.load_pem_private_key(
TypeError: Password was given but private key is not encrypted.
[2025-05-22T01:37:23.919+0530] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=elt_sales_stream_transformation, task_id=transform_sales_stream, run_id=manual__2025-05-21T20:07:20.807512+00:00, execution_date=20250521T200720, start_date=20250521T200723, end_date=20250521T200723
[2025-05-22T01:37:23.931+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-22T01:37:23.932+0530] {standard_task_runner.py:124} ERROR - Failed to execute job 128 for task transform_sales_stream (Password was given but private key is not encrypted.; 168944)
Traceback (most recent call last):
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/providers/snowflake/operators/snowflake.py", line 420, in execute
    self.query_ids = self._hook.execute_query(
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/providers/snowflake/hooks/snowflake_sql_api.py", line 140, in execute_query
    conn_config = self._get_conn_params
  File "/usr/lib/python3.10/functools.py", line 981, in __get__
    val = self.func(instance)
  File "/home/pranav_shirali/airflow-project/airflow-env/lib/python3.10/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 315, in _get_conn_params
    p_key = serialization.load_pem_private_key(
TypeError: Password was given but private key is not encrypted.
[2025-05-22T01:37:23.971+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-05-22T01:37:23.980+0530] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-22T01:37:23.980+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
